src/Gtk/Avalonia.Gtk/WindowImpl.cs

```diff
@@ -60,20 +60,18 @@ protected override void OnRealized ()
             _imContext.ClientWindow = this.GdkWindow;
         }
 
-        public Size ClientSize
+        public virtual Size ClientSize
         {
             get
             {
                 int width;
                 int height;
                 GetSize(out width, out height);
                 return new Size(width, height);
-            }
-
-            set
-            {
-                Resize((int)value.Width, (int)value.Height);
-            }
+            }	
+            set 
```

Comment:
I'm not a fan of the empty set clause. Can you explain your rationale (i.e. what breaks if it isn't empty)?